include/sdbus-c++/AdaptorInterfaces.h

```diff
@@ -107,6 +107,14 @@ namespace sdbus {
         {
         }
 
+        /*!
+         * @brief return object path of the underlying DBus object
+         */
+        const std::string& getObjectPath() const
```

Comment:
I'd prefer that we define this function at the end of the class, after `unregisterAdaptor`. This method is not so essential as the other ones for class operation, and these ones I would like to keep up in the class to be visible immediately. Would you please move it there?

Reply:
Done